gnome-image-installer/pages/install/gis-scribe.c

```diff
@@ -228,6 +230,11 @@ gis_scribe_set_property (GObject      *object,
       self->convert_to_mbr = g_value_get_boolean (value);
       break;
 
+    case PROP_GPG_PATH:
+      g_free (self->gpg_path);
+      self->gpg_path = g_value_dup_string (value);
```

Comment:
Nitpick: Maybe add some assertions that the path is non-empty and absolute? I normally add such assertions in the `constructed` vfunc, since it’s guaranteed to be called after all construct-only properties are set, regardless of whether a value is actually provided for the property (or the default value is used).

Reply:
There's a non-NULL assertion in `constructed`. I'll add a check for absolution, and make it hold true.